index.js

```diff
@@ -1,92 +1,99 @@
+     currentThirstLevel -= 1
+  }
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
-
-
-
+console.log("%c----------", "color: red")
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
-
-// console.log(sameSameButDifferent(5, 5)) 
+console.log("%cQuestion 4", "color: red")
+const sameSameButDifferent = (num, maybenum) => {
+  if (num === maybenum) {
+    return "same same"
+  } else if (num == maybenum) {
+    return "same same(but different)"
+  }
+  else {
+    return "different"
+  }
+}
+console.log(sameSameButDifferent(5, 5))
 // // => "same same"
-
-// console.log(sameSameButDifferent(123, "123")) 
+console.log(sameSameButDifferent(123, "123"))
 // // => "same same (but different)"
-
-// console.log(sameSameButDifferent(5, 7)) 
+console.log(sameSameButDifferent(5, 7))
 // // => "different"
-
-// console.log(sameSameButDifferent(123, "122")) 
+console.log(sameSameButDifferent(123, "122"))
 // // => "different"
-// console.log("%c----------", "color: red") 
-
-
-
+console.log("%c----------", "color: red")
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+console.log("%cQuestion 5", "color: red")
+const updateGrade = (student, grade) => {
+  return student.grade = grade
+}
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)
 // // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red")
-
-
-
+console.log("%c----------", "color: red")
 // ***** Question 6 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
+console.log("%cQuestion 6", "color: red")
+const users = [
+  {
+    name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
+  },
+  {
+    name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
+  }
+]
+const printNameAndPhones = (user) => {
+  for (i = 0; i < users.length; i++){
+    console.log(users[i]["name"])
+    console.log(users[i]["phones"]["cell"])
+    console.log(users[i]["phones"]["office"])
+  }
+}
+printNameAndPhones(users)
 // // => "Duane"
 // => "Cell: 555-123-4567"
 // => "Office: 555-456-7890"
 // => "Liza"
 // => "Cell: 555-234-5678"
 // => "Office: 555-567-1234"
-// console.log("%c----------", "color: red") 
-
-
+// console.log("%c----------", "color: red")
 // ***** Callbacks *****
-
 function myMap(array, callback) {
```

Comment:
Da-Mell, you were on the way to the solution. Here is my solution, first we create the function triple.

```
const triple = num => {
  return num * 3;
}
```
which is the same thing you have, but it is not expecting an array as an argument, but a single number.